setup.py

```diff
@@ -74,7 +74,7 @@
         "Topic :: System :: Monitoring",
         ],
     install_requires=[
-        'requests<=2.10.0',
+        'requests>=0.12.1,<=2.10.0',
```

Comment:
Actually, yeah I can do that. I've already done some conditional stuff for the tests_require part.